README.md

```diff
@@ -0,0 +1,14 @@
+# pairviewer
+
+An interactive [D3.js](https://d3js.org/) graph of [Apertium language pairs](http://wiki.apertium.org/wiki/List_of_language_pairs) you can search, filter, and drag around. [Online demo](http://ilazki.thinkgeek.co.uk/~firespeaker/pairviewer/apertium.html).
+
+The relevant Apertium wiki page (along with local setup instructions!) is available [**here**](http://wiki.apertium.org/wiki/Pair_viewer).
```

Comment:
Links with the text here are not ideal. Instead just say something like "See the relevant Apertium wiki page for local setup instructions" with the wiki page part linked.